.gitignore

```diff
@@ -11,5 +11,5 @@ build
 
 # misc
 .DS_Store
-.env
 npm-debug.log
+.firebase/**
```

Comment:
FYI if you want to ignore a whole folder, you can just use `.firebase/`

Reply:
👍